framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -232,5 +232,52 @@ int cfg_get_sta_iface(const char iface[BPL_IFNAME_LEN], char sta_iface[BPL_IFNAM
     return RETURN_OK;
 }
 
+int cfg_get_hostap_iface(int32_t radio_num, char hostap_iface[BPL_IFNAME_LEN])
+{
+    if (!hostap_iface) {
+        MAPF_ERR("cfg_get_hostap_iface: invalid input: hostap_iface is NULL");
+        return RETURN_ERR;
+    }
+
+    if (radio_num < 0) {
+        MAPF_ERR("cfg_get_hostap_iface: invalid input: radio_num < 0");
+        return RETURN_ERR;
+    }
+
+    std::string iface_str("wlan" + std::to_string(radio_num));
```

Comment:
Maybe the interface prefix should not be hard-coded.
Can be moved to agent config file or the UCI DB.
@tomereli?

Reply:
Also, where do you actually look for the UCI key?

Reply:
This is indeed wrong and beats the purpose - we don't want any dependency on the interface name. It should be set in /etc/config/prplmesh once and that's it - no radio_num dependency should be there.

Reply:
this is Linux BPL - so no /etc/config/prplmesh
in order to read this configuration from the prplmesh_platform_db file, I need to add infrastructure for reading non-flat radio configuration from the prplmesh_platform_db file.

do we want this to be part of this PR?

Reply:
talked with @tomereli and since this is the Linux BPL - decided to leave it as is.